product_landing_page/index.css

```diff
@@ -203,6 +203,40 @@ body {
   background-color: darkorange;
 }
 
+.footer {
+  width: 75%;
+  background: #ddd;
+  display: flex;
+  flex-direction: column;
+}
+
+.footer__link {
+  color: #333;
+  display: flex;
+  text-decoration: none;
+  padding: .5rem 0 0 .5rem;
+  margin-right: 1.5rem;
+}
+
+.footer__link:hover {
+  transition: 300ms;
+  opacity: 0.4;
```

Comment:
Wydaje mi się, że to jest zbyt niska wartość `opacity`, tekst staje się słabo czytelny, lepiej zmienić na coś subtelniejszego.